apps/src/lib/ui/ConfirmEnableMakerDialog.story.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {ConfirmEnableMakerDialog} from './ConfirmEnableMakerDialog';
+
+export default storybook => {
+  return storybook
+      .storiesOf('ConfirmEnableMakerDialog', module)
+      .add('overview', () => {
+        return (
+          <ConfirmEnableMakerDialog
+            isOpen={true}
```

Comment:
I don't know about you, but I really like the xml flag syntax where you just say `isOpen` without `isOpen={true}`. up to you though.